app/ui/page_inputs.go

```diff
@@ -189,6 +189,12 @@ func (p *MangaPage) setHandlers(cancel context.CancelFunc) {
 		return event
 	})
 
+	p.Table.SetSelectionChangedFunc(func(row, _column int) {
```

Comment:
A comment on what this binding does would be useful. See line 182 of same file for example.